src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/LocalBuilderImpl.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Reflection.Emit
+{
+    internal sealed class LocalBuilderImpl : LocalBuilder
+    {
+        #region Private Data Members
+        private readonly int _localIndex;
+        private readonly Type _localType;
+        private readonly MethodInfo _method;
+        private readonly bool _isPinned;
+        #endregion
+
+        #region Constructor
+        internal LocalBuilderImpl(int index, Type type, MethodInfo method, bool isPinned)
+        {
+            _isPinned = isPinned;
+            _localIndex = index;
+            _localType = type;
+            _method = method;
+        }
+        #endregion
+
+        #region LocalVariableInfo Override
+        public override bool IsPinned => _isPinned;
+        public override Type LocalType => _localType;
+        public override int LocalIndex => _localIndex;
+        public override MethodInfo Method => _method;
```

Comment:
Should be from `LocalBuilder`.

Reply:
Sorry did not understood, the abstract `LocalBuilder` doesn't have this field.

Reply:
It is defined in https://github.com/dotnet/runtime/pull/93809/files#diff-b17fa465e3ae3bd103760932136e6317b718ce3d30f551b3976e7ee3c9070d45?